CI Failing: test_geopandas

https://travis-ci.org/pandas-dev/pandas/jobs/653274362?utm_medium=notification&utm_source=github_status

cc @jorisvandenbossche 


```
=================================== FAILURES ===================================
________________________________ test_geopandas ________________________________
[gw0] linux -- Python 3.6.10 /home/travis/miniconda3/envs/pandas-dev/bin/python
    @pytest.mark.filterwarnings("ignore:can't resolve:ImportWarning")
    def test_geopandas():
    
>       geopandas = import_module("geopandas")  # noqa
pandas/tests/test_downstream.py:112: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
pandas/tests/test_downstream.py:20: in import_module
    return importlib.import_module(name)
../../../miniconda3/envs/pandas-dev/lib/python3.6/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:994: in _gcd_import
    ???
<frozen importlib._bootstrap>:971: in _find_and_load
    ???
<frozen importlib._bootstrap>:955: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:665: in _load_unlocked
    ???
<frozen importlib._bootstrap_external>:678: in exec_module
    ???
<frozen importlib._bootstrap>:219: in _call_with_frames_removed
    ???
../../../miniconda3/envs/pandas-dev/lib/python3.6/site-packages/geopandas/__init__.py:5: in <module>
    from geopandas.io.file import read_file  # noqa
../../../miniconda3/envs/pandas-dev/lib/python3.6/site-packages/geopandas/io/file.py:6: in <module>
    import fiona
../../../miniconda3/envs/pandas-dev/lib/python3.6/site-packages/fiona/__init__.py:83: in <module>
    from fiona.collection import BytesCollection, Collection
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
    import logging
    import os
    import warnings
    
    from fiona import compat, vfs
>   from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
E   ImportError: libkea.so.1.4.7: cannot open shared object file: No such file or directory
../../../miniconda3/envs/pandas-dev/lib/python3.6/site-packages/fiona/collection.py:9: ImportError

```

--- a/pandas/tests/test_downstream.py
+++ b/pandas/tests/test_downstream.py
@@ -107,7 +107,6 @@ def test_pandas_datareader():
 
 # importing from pandas, Cython import warning
 @pytest.mark.filterwarnings("ignore:can't resolve:ImportWarning")
-@pytest.mark.skip(reason="Anaconda installation issue - GH32144")
 def test_geopandas():
 
     geopandas = import_module("geopandas")  # noqa

